pom.xml

```diff
@@ -130,6 +142,12 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>0.7.8</version>
+                <configuration>

```

Comment:
This is not needed. Your `model` and `dto` classes should be tested withing testing other layers